diff -urbB tfel-TFEL-3.1.1.orig/CMakeLists.txt tfel-TFEL-3.1.1/CMakeLists.txt
--- tfel-TFEL-3.1.1.orig/CMakeLists.txt 2018-03-07 09:01:26.000000000 +0100
+++ tfel-TFEL-3.1.1/CMakeLists.txt      2018-12-18 14:25:47.798261457 +0100
@@ -256,7 +256,8 @@
   message(STATUS "python include path   ${PYTHON_INCLUDE_DIRS}")
   message(STATUS "python libraries path ${PYTHON_LIBRARY_PATH}")
   message(STATUS "python library        ${PYTHON_LIBRARY}")
-  find_package(PythonInterp ${PYTHONLIBS_VERSION_STRING} REQUIRED)
+  string(REGEX REPLACE "[a-z]+.*$" "" PYTHONLIBS_VERSION_CLEANED ${PYTHONLIBS_VERSION_STRING})
+  find_package(PythonInterp ${PYTHONLIBS_VERSION_CLEANED} REQUIRED)
   message(STATUS "python interpreter    ${PYTHON_EXECUTABLE}")
 endif(enable-python OR enable-python-bindings)
